core/blockchain.go

```diff
@@ -1800,8 +1814,8 @@ func (bc *BlockChain) insertChain(chain types.Blocks, verifySeals bool) (int, er
 // switch over to the new chain if the TD exceeded the current chain.
 func (bc *BlockChain) insertSideChain(block *types.Block, it *insertIterator) (int, error) {
```

Comment:
does this methos makes sense now?

Reply:
Removed, also with the reorg method

Reply:
Re added but just for the case of importing a chain, with previously pruned blocks